src/handlers.js

```diff
@@ -278,3 +279,14 @@ async function isCollaborator({login, organization, repository, sha, context}) {
   });
   return false;
 }
+
+async function createTasksInGroup(queue, graphConfig) {
+  let taskGroupId = null;
+  let created = [];
+  for (let task of graphConfig.tasks) {
+    taskGroupId = task.task.taskGroupId;
+    created.push(queue.createTask(task.taskId, task.task));
+  }
+  await Promise.all(created);
```

Comment:
this is a `Promise.all(graphConfig.map(t => queue.createTask(t.taskId, t.task)));`